### PR TITLE
Update mao.yaml

### PR DESCRIPTION
**Pull request by theethaj**
Update mao.yaml to match with v1.1.0

### CODE REVIEW

File: mao.yaml [hunk 1]
```diff
@@ -1607,3 +1622,14 @@ owl:ObjectProperty:
...
+owl:Individual:
+  family:Female:
+    rdf:type:
+      - Gender
+  family:Male:
+    rdf:type:
+      - Gender
+  family:Non-binary:
+    rdf:type:
+      - Gender
```
Comment (MaiMee1) [inline mao.yaml:1635]: No newline at EOF!

Comment (theethaj) [reply mao.yaml]: It's now corrected, thanks. Please review again.

**Review by th-bunratta — CHANGES_REQUESTED** [review]
The following properties are missing from object property definitions, but classes are referring to them.
- `mao:hasStudio`
- `mao:locatedIn`

Please define them.

**Comment by NutthanichN** [comment]
> 
> 
> The following properties are missing from object property definitions, but classes are referring to them.
> 
>     * `mao:hasStudio`
> 
>     * `mao:locatedIn`
> 
>     * `mao:MovieCast`
> 
>     * `mao:AggregateRatingStatement`
> 
> 
> Also when defining `hasValue` restriction as in `FilmEligibleForAward and ('satisfies criterion' only OscarCriterion) and ('violates criterion' only (not (OscarCriterion))) and ('eligible for' value Oscars)`, the specs will need to know what class the individual `Oscars` is from. Better to define it in the `mao:Individual` section as well.
> 
> Please define them.

Those properties except `mao:locatedIn` are outdated so I removed them and defined only `mao:locatedIn` in the `owl:ObjectProperty` section.

I'll let you know if I finish defining those individuals in the `owl:Individual` section.

**Pull request merged**